source/MultiGrid/MultiGrid.js

```diff
@@ -732,16 +773,31 @@ export default class MultiGrid extends React.PureComponent {
         }
         columnWidth={this._columnWidthRightGrid}
         deferredMeasurementCache={this._deferredMeasurementCacheTopRightGrid}
-        height={this._getTopGridHeight(props)}
+        height={gridHeight}
         onScroll={enableFixedRowScroll ? this._onScrollLeft : undefined}
         ref={this._topRightGridRef}
         rowCount={fixedRowCount}
         scrollLeft={scrollLeft}
-        style={this._topRightGridStyle}
+        style={style}
         tabIndex={null}
-        width={this._getRightGridWidth(props)}
+        width={width}
       />
     );
+
+    if (hideTopRightGridScrollbar) {
+      return (
+        <div
+          style={{
+            ...this._topRightGridStyle,
+            height,
+            width,
+            overflowX: 'hidden',
```

Comment:
I don't think you need to wrap it in another div

If you need to. you can just modify `this._topRightGridStyle`

```js
this._topRightGridStyle = {
   ...this._topRightGridStyle,
   height,
   width, 
   overflowX: 'hidden'
}
```

Reply:
Yes. We need this extra wrapper so we can hide the scrollbar with a hack.

Reply:
Ah I see. Thanks

Reply:
@wuweiweiwu , are you waiting for some change here?